substrate/frame/multisig/src/lib.rs

```diff
@@ -665,6 +691,25 @@ impl<T: Config> Pallet<T> {
 		signatories.insert(index, who);
 		Ok(signatories)
 	}
+
+	pub(crate) fn ensure_updated(id: &T::AccountId, call_hash: &[u8; 32], threshold: u16) -> bool {
+		#[allow(deprecated)]
+		let r = match Multisigs::<T>::take(id, call_hash) {
+			Some(r) => r,
+			None => return false,
+		};
+		T::Currency::unreserve(&r.depositor, r.deposit);
+		// take consideration
+		let Ok(ticket) =
+			T::Consideration::new(&r.depositor, Footprint::from_parts(1, threshold as usize))
+				.defensive_proof("Unexpected inability to take deposit after unreserved")
+		else {
+			return true
```

Comment:
if this returns early, the new multisig is not added to `MultisigsFor`. should we just defensive log but continue anyways to not break the multisig?

Reply:
Well, multisig can't exist in the system if we were not able to take the consideration. I can throw an error in this case though.

Reply:
Wouldn't it be bad ux for a multisig to just vanish?

This shouldn't be an issue if we don't try re-reserving different amounts, though

Reply:
It won't vanish but the migration would fail hinting the user to add funds if needed.

Reply:
I don't see where it fails, maybe I misunderstand but I see it returns true in this conditional if the consideration fails to create